test_python_generator.py

```
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
+"""
+Simple test script for Python generator
+"""
+
+import sys
+import os
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), 'src'))
+
+from ssis_migrator.generators.python_generator import PythonScriptGenerator
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)